src/components/Tree/TreeItem.tsx

```diff
@@ -0,0 +1,143 @@
+import * as _ from 'lodash'
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+
+import Tree from './Tree'
+import TreeTitle from './TreeTitle'
+import { defaultBehavior } from '../../lib/accessibility'
+import { Accessibility } from '../../lib/accessibility/types'
+
+import {
+  AutoControlledComponent,
+  childrenExist,
+  customPropTypes,
+  createShorthandFactory,
+} from '../../lib'
+import { ShorthandRenderFunction, ShorthandValue } from 'utils'
+import {
+  commonUIComponentPropTypes,
+  childrenComponentPropTypes,
+  contentComponentPropsTypes,
+} from '../../lib/commonPropTypes'
+import {
+  UIComponentProps,
+  ChildrenComponentProps,
+  ContentComponentProps,
+} from '../../lib/commonPropInterfaces'
+
+export interface TreeItemProps
+  extends UIComponentProps<any, any>,
+    ChildrenComponentProps,
+    ContentComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   * @default defaultBehavior
+   */
+  accessibility?: Accessibility
+
+  /** Initial open value. */
+  defaultOpen?: boolean
```

Comment:
I am not sure about exact use case where it might be necessary. If it is about specifying default value for `open` prop, then I would suggest to use `defaultProps` for that. 

At the same time, it seems that this prop is not used _anywhere_ now - would suggest to remove it then.